src/components/tasks/card/card.module.scss

```diff
@@ -364,6 +380,14 @@ $progressTextAnimation: #9babb8;
     flex-direction: column;
     justify-content: center;
     align-items: center;
+    @media (max-width: 28rem) {
+        padding: 1rem;
+    }
+
+    // 368px
+    // @media (max-width: 23rem) {
+    //     padding: 0.5rem ;
```

Comment:
Please remove this piece of commented code

Reply:
Removed comments